### PR TITLE
Feat/dumpsh

### PR DESCRIPTION
**Pull request by paulineribeyre**
- add dump.sh script
- add PR template
- update cheat sheet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,7 +149,14 @@ after you update some code in order to see changes without having to rebuild all
 ### Running Docker Compose on a Remote Machine
 To run Docker Compose on a remote machine, modify the `hostname` field in `fence-config.yaml`, `peregrine_creds.json`, and `sheepdog_creds.json` in the `Secrets` directory.
 
-* * *
+### Dumping config files and logs
+
+If you are encountering difficulties while setting up Docker Compose and need help from the Gen3 team, you can use the `dump.sh` script to create a zip file of your configuration and current logs, which you can share to get help.
```
Comment (Avantol13) [inline README.md]: should add a note that the configurations can contain *sensitive* credentials maybe?

Comment (paulineribeyre) [reply README.md]: mm yeah 👍

File: README.md [hunk 2]
```diff
@@ -149,7 +149,21 @@ after you update some code in order to see changes without having to rebuild all
...
+bash dump.sh
+```
+Note that if docker-compose is not running, the logs will be empty.
+
+The following configuration files will be included:
+* docker-compose.yml
+* user.yaml
+* any file ending with "settings" or "config"
+
+Make sure to **remove sensitive credentials** that should not be shared.
```
Comment (Avantol13) [inline README.md]: but this zips everything up first, so maybe we should have a `prep.sh` which moves everything to a directory and then tell them to remove sensitive info and then run `dump.sh` and it'll zip it all? idk, this is kinda awkward if we're asking them to remove creds b/c we don't want them to remove them from the Secrets folder or they'll break their stuff. and kinda weird to zip it all, tell them to unzip, change stuff, and rezip

Comment (paulineribeyre) [reply README.md]: i was thinking they can remove the creds, save, run the script, and put the creds back

Comment (paulineribeyre) [reply README.md]: but in any case it will be awkawrd

**Pull request merged**